src/utils/unique-string-array.js

```diff
@@ -24,6 +24,10 @@ export class UniqueStringArray {
     return this._array[index];
   }
 
+  hasString(s: string): boolean {
```

Comment:
I was mutating the stringTable and causing test churn. This felt like the cleaner solution.